### ISSUE TITLE
Midjouney Blocked for using bot!

### ISSUE THREAD

**Issue by aryan1107** [imagineapi/imagineapi#14]
I get this error message

Midjourney message: You have a pending moderation message. Please acknowledge it before using Midjourney further.
Your account has been temporarily blocked due to suspicious activity. Accessing Midjourney via third-party tools or scripting is strictly forbidden and is grounds for a permanent ban..

*Please review Midjourney moderation guidelines [here](https://docs.midjourney.com/docs/community-guidelines)*

**Comment by hgezim** [comment]
In the [cloud plan](https://www.imagineapi.dev/pricing) we allow you to use multiple bots to split load.

But yeah bans can happen when you have a heavy load.

**Comment by aChase55** [comment]
@hgezim  Hi, I'm facing the same issue. I don't see how to add a second bot anywhere. I see an option under flows in the right hand column, that says add mid journey bot. I clicked that and got a message saying I'd be charged for additional bots (how much? who knows?) I clicked okay, and the pop up closed, saying it was successful. But I still don't see how to actually connect a second account, and I have no idea how much I was charged
